tools/mo/openvino/tools/mo/convert.py

```diff
@@ -0,0 +1,20 @@
+# Copyright (C) 2018-2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+from collections import namedtuple
+from openvino.tools.mo.convert_impl import _convert
+
+InputCutInfo = namedtuple("InputInfo", ["name", "shape", "type", "value"])
+LayoutMap = namedtuple("LayoutMap", ["source_layout", "target_layout"])
+
+
+def convert(**args):
+    """
+    Converts the model from original framework to OpenVino Model.
+
+    Run convert(help=true) to list available parameters.
+
```

Comment:
May I ask you to add description what kind of arguments can be here please? I know that the documentation is in separate one but for developers this is important now to see what arguments we are going to handle.
It is difficult maintainable currently.

Reply:
@slyalin asked not to put the description here to avoid duplication with cli parser. So the temporary solution to provide help=True argument. The full solution with will be covered in 89898.

Reply:
I asked not to duplicate description. If you can put the description here and then reuse it in the cli parser would be nice. I agree that it shouldn't block us and better to cover in a separate ticket.